File: JavaScript/SelectionSort/src/Author.js

  function Author(_firstName, _lastName) {

    var self = this;
    self.firstName = _firstName;
    self.lastName = _lastName;

    function compareTo(_author) {
      return self.lastName.localeCompare(_author.getLastName());
    }

    function getFirstName() {
      return self.firstName;
    }

    function getLastName() {
      return self.lastName;
    }

    return {
      compareTo: compareTo,
      getFirstName: getFirstName,
      getLastName: getLastName
    }
  }

  module.exports = Author;

File: Java/LinkedList/src/domain/Author.java
package domain;

public class Author{
	
	private String firstName;
	private String lastName;
	
	public Author(String _firstName, String _lastName) {
		this.firstName = _firstName;
		this.lastName = _lastName;
	}
	
	public String getFirstName() {
		return this.firstName;
	}
	
	public String getLastName() {
		return this.lastName;
	}
}
